app/javascript/lib/imperial.js

```diff
@@ -2332,7 +2336,8 @@ export default class Imperial {
     let bonus = this.playerBonusBeforeUnitMaintenanceCosts(nationName, taxes);
 
     if (this.baseGame === 'imperial2030' || this.baseGame === 'imperialAsia') {
-      const treasuryAmountAfterMaintenanceCosts = this.nations.get(nationName).treasury + taxes - this.unitMaintenanceCosts(nationName);
+      const treasuryAmountBeforeMaintenanceCosts = this.nations.get(nationName).treasury + taxes;
+      const treasuryAmountAfterMaintenanceCosts = treasuryAmountBeforeMaintenanceCosts - this.unitMaintenanceCosts(nationName);
```

Comment:
This is a great clarification! 🙏

Reply:
Mostly to keep eslint from complaining about line length